apps/omg_watcher_info/lib/omg_watcher_info/http_rpc/client.ex

```diff
@@ -34,14 +34,28 @@ defmodule OMG.WatcherInfo.HttpRPC.Client do
   @spec get_block(binary(), binary()) :: response_t()
   def get_block(hash, url), do: call(%{hash: Encoding.to_hex(hash)}, "block.get", url)
 
+  @doc """
+  Gets supported fees
+  """
+  @spec get_fees(map(), binary()) ::
```

Comment:
```suggestion
  @spec get_fees(map(), binary()) :: response_t()
```

Then we don't need the rest of the return signature? I notice it's the more generic `map()` here but I think they're equal?

Reply:
In this case they will not be equal, the keys will be strings and not atom